### DISCUSSION TITLE
Is there a way to position a fixed data label in topmost of entire stack for a stacked bar graph

### DISCUSSION THREAD

**Discussion by albert-kp** [recharts/recharts#4853] [Q&A]
![image](https://github.com/user-attachments/assets/725849d5-fedd-4335-a29c-b28eb991df6a)

I need to position a fixed data label in top of the topmost stack of stacked bar graph. based on data some of the bars may have lesser stacks so need to correctly place the data label to top most item. Is there a way to achieve this

**Comment by ckifer** [comment]
Does `LabelList` work?

https://codesandbox.io/p/sandbox/stacked-bar-chart-forked-7jf3sq
